notebooks/working/6.0-mab-estimate-causal-effect.md

```diff
@@ -2200,15 +2200,15 @@ for sim_size, number in zip(simulation_sizes, sim_number):
     ############ Drive Alone ############
```

Comment:
Can you include a comment under each of the `######### Mode Name #######` lines that alerts readers to the effect of using `2 * regression_parameter` in the model for the downstream nodes? In particular, the models for the downstream nodes no longer produce data with the exact means as the real data, but the data is exaggerated to clearly illustrate the causal effect differences that can possibly arise in real life.